.gitignore

```diff
@@ -1,2 +1,3 @@
 .sass-cache
-node_modules
\ No newline at end of file
+node_modules
+.DS_Store
```

Comment:
Since `.DS_Store` is an environmental file, rather than a project file, you might want to add that into your global gitignore config instead.